src/main/java/TaskList.java

```diff
@@ -0,0 +1,112 @@
+import java.util.ArrayList;
```

Comment:
The TaskList class have a lot of functionalities and it looks quite complete. Nice!